src/Pages/Auth/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect } from "react";
+import logo from "../../assets/budget-it.png";
+import { TimelineLite } from "gsap";
+import { Link, Redirect, Route, Switch } from "react-router-dom";
+import Register from "./Register.jsx";
+import Login from "./Login.jsx";
+
+const Auth = () => {
+  let tl = new TimelineLite({ paused: true });
+
+  useEffect(() => {
+    tl.fromTo(
+      ".text1",
+      2,
+      {
+        width: "0",
+      },
+      {
+        width: "260px",
+        ease: "steps(20)",
+      },
+      0
+    );
+
+    tl.fromTo(
+      ".text1",
+      0.5,
+      { borderRightColor: "rgba(255,255,255,0.8)" },
+      {
+        repeat: -1,
+        ease: "steps(14)",
+        borderRightColor: "rgba(255,255,255,0)",
+      },
+      0
+    ).fromTo(
+      ".text2",
+      2,
+      {
+        width: "0",
+      },
+      {
+        width: "340px",
+        ease: "steps(20)",
+        delay: 2,
+      },
+      0
+    );
+
+    tl.fromTo(
+      ".text2",
+      0.5,
+      { borderRightColor: "rgba(255,255,255,0.8)" },
+      {
+        repeat: -1,
+        ease: "steps(14)",
+        borderRightColor: "rgba(255,255,255,0)",
+      },
+      0
+    );
+    tl.play();
+  }, []);
+
+  if (localStorage.user) {
```

Comment:
I didn't know about this. i had to ask my boyfriend to explain. Wooow. You are doing great